etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
+KUBECTL=${KUBECTL:-kubectl}
+PACHCTL=${PACHCTL:-pachctl}
+RM=${RM:-rm}
+SLEEP=${SLEEP:-sleep}
+TAR=${TAR:-tar}
+WC=${WC:-WC}
+WGET=${WGET:-wget}
```

Comment:
I'm not aware of any platform differences for wget, could you elucidate?

Reply:
wget is a tool that would need to be installed, and this script may not run where it's in the PATH.  I don't want to rely on that, and I don't want to require that a PATH be changed to run this for tools that must be installed, for security reasons.  That's why I allow folks to specify the install location of tools that aren't part of a standard buid.